starter-code/src/components/Footer.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import "./../styles/White.css";
+import "./../styles/App.css";
+
+const Footer = () => {
+    return (
+        <footer>
+            <div className="footer-item">
+                <img src="./images/icon1.png" alt="icon"/>
+                <h2>Declarative</h2>
+                <p>React makes it
+                    <br/>painless to create
+                    <br/>interactive UIs.
+                </p>
+            </div>
```

Comment:
You are repeating yourself on this part 4 times.
It could become a component ressembling something like below.
It would alow you to reuse it somewhere else in your application.
```
import React from "react";


const InfoBox = props => {
  return (
    <div className="footer-item">
      <img src={props.image} alt="icon" />
      <h2>{props.title}</h2>
      <p>{props.subTitle}</p>
    </div>
  );
};

export default InfoBox;

```